### PR TITLE
ci: revert commits trigger a patch version

### PR DESCRIPTION
**Pull request by simonrose121**
## Description

Music year: 1950

- Adds semantic release configuration for the `revert:` commit type to trigger a patch release (by default this requires a commit description, which we've decided we don't need due to our existing process and that we're normally doing reverts in a rush!)
- I've written up some [accompanying documentation](https://www.notion.so/oaknationalacademy/OWA-rollback-guide-b47ce1370d464ce4a48e1e8b8ed4306d?pvs=4) for how to use this to rollback a Netlify deployment.

## Issue(s)

[ENG-34](https://www.notion.so/oaknationalacademy/Update-the-semantic-versioning-config-in-OWA-to-treat-revert-commits-as-needing-to-increment-the-pat-195499f3110148e4bca9eb15f862f472?pvs=4)

## How to test

- [x] Test on forked repo.

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ module.exports = {
         releaseRules: [
           { type: "chore", release: "patch" },
           { type: "refactor", release: "patch" },
+          { type: "revert", release: "patch" },
```
[the remark this answers is not in the document]
Comment (simonrose121) [reply release.config.js:13]: @njblore I think you're right, it also looks like there's a better way to do the revert commit messages: https://semantic-release.gitbook.io/semantic-release/support/faq#how-can-i-revert-a-release

I'll close this PR as there's no code change and update the docs 🙏

[the remark this answers is not in the document]
Comment (simonrose121) [reply release.config.js:13]: @njblore I've given it a test on my ai-beta test repo (with the same sem release config as OWA) and it doesn't patch by default: https://github.com/oaknational/ai-beta-release-process-test/actions/runs/10262727751/job/28393007703

I'm not sure what that line in the default config is doing!

I've reopened the PR.

**Review by njblore — APPROVED** [review]

File: release.config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (simonrose121) [reply release.config.js:13]: It's [a known issue](https://github.com/semantic-release/commit-analyzer/issues/626), the `revert:` commit type requires a description.

We've decided to drop that requirement (by accepting this change), as it better fits our current process and we tend to be reverting in a rush!

**Pull request merged**